grakn-core/src/main/java/ai/grakn/GraknSession.java

```diff
@@ -64,9 +62,8 @@
     /**
      * Closes the main connection to the graph. This should be done at the end of using the graph.
      *
-     * @throws GraknTxOperationException when more than 1 transaction is open on the graph
      */
-    void close() throws GraknTxOperationException;
```

Comment:
I think it's good to keep `RuntimeException`s in the Javadocs (but not the signatures). An example of this [here](https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#sort(java.util.List)) to prove I'm not crazy.

Reply:
Sonar does not like it and neither do I to be honest. It will be do difficult to maintain as internal implementation changes. So I would say leave it removed but can put it back if you feel strongly about it.

Reply:
Are you sure it doesn't like it in the Javadoc? Most libraries I can find includes `RuntimeException`s in the Javadoc.

To be clear, I agree with removing them from the method signature - since that does nothing.

Reply:
I would say all the APIs should have `@throws` for runtime exceptions or none of them should. This middle ground is kind of silly and does not provide much information. 

Technically nearly every method on the core api can throw `GraknTxOperationException`. So we should be adding it everywhere.

Sorry but I am being a stickler here as well. If it's hard to maintain and the middle ground is meaningless then we should remove it.